app/usecases/images.py

```diff
@@ -0,0 +1,89 @@
+import io
+import logging
+from enum import Enum
+
+from PIL import Image
+
+from app.adapters import rekognition
+from app.adapters import s3
+from app.errors import Error
+from app.errors import ErrorCode
+
+DISALLOWED_MODERATION_LABELS = [
+    "Explicit Nudity",
+    "Suggestive",
+    "Violence",
+    "Visually Disturbing",
+]
```

Comment:
Going with a whitelist approach here, seems a bit safer

Reply:
This list is not yet finalized, will do another round of confirming we have all the desirable behaviour before merge

Reply:
Here are the labels: https://docs.aws.amazon.com/rekognition/latest/dg/moderation.html#moderation-api